unit_scaling/_modules.py

```diff
@@ -181,6 +182,73 @@ def forward(self, input: Tensor) -> Tensor:
         return U.linear_readout(input, self.weight, self.bias, self.constraint)
 
 
+@inherit_docstring(
+    short_description=(
+        "Applies a **unit-scaled** 1D convolution to the incoming data."
+        "\nNote that this layer sets :code:`bias=False` by default."
+    ),
+    add_args=[binary_constraint_docstring],
+)
+class Conv1d(nn.Conv1d):
+    def __init__(
+        self,
+        in_channels: int,
+        out_channels: int,
+        kernel_size: int,
+        stride: int = 1,
+        padding: Union[str, int] = 0,
+        dilation: int = 1,
+        groups: int = 1,
+        bias: bool = False,
+        padding_mode: str = "zeros",
+        device: Any = None,
+        dtype: Any = None,
+        constraint: Optional[str] = "to_output_scale",
+        weight_mup_type: MupType = "weight",
+    ) -> None:
+        super().__init__(
+            in_channels,
+            out_channels,
+            kernel_size,
+            stride,
+            padding,
+            dilation,
+            groups,
+            bias,
+            padding_mode,
+            device,
+            dtype,
+        )
+        self.kernel_size = kernel_size  # type:ignore[assignment]
+        self.stride = stride  # type:ignore[assignment]
+        self.padding = padding  # type:ignore[assignment]
+        self.dilation = dilation  # type:ignore[assignment]
+        self.constraint = constraint
+        self.weight = Parameter(self.weight.data, mup_type=weight_mup_type)
+        if self.bias is not None:
+            self.bias = Parameter(self.bias, mup_type="bias")
```

Comment:
(I notice this is in `Linear` too, so my bad) - perhaps this should be `self.bias.data` for consistency, although I suspect it doesn't matter.

Reply:
Fixed!